.github/workflows/mediasoup-pack.yaml

```diff
@@ -0,0 +1,125 @@
+name: mediasoup-pack
+
+on:
+  push:
+    tags:
+    - v*  # Only build for tags that match new versions of `npm` packages
```

Comment:
We're not using this format for tags, so it will definitely have to change.

Reply:
No issues :-) My intention was to automate the process based on how `npm` works, but that can be changed.